code/lib/cli-storybook/src/doctor/getIncompatibleStorybookPackages.ts

```diff
@@ -127,6 +153,22 @@ export const getIncompatiblePackagesSummary = (
       'For more on Storybook 8 compatibility, see the linked GitHub issue:',
```

Comment:
If you're doing this work for Storybook 9, you should probably update this part too

Reply:
There is a ticket for this in the breaking changes workstream